addons/intelitems/CfgMagazines.hpp

```diff
@@ -20,6 +22,7 @@ class CfgMagazines {
 
     class XGVAR(document): GVAR(base) {
         author = ECSTRING(common,ACETeam);
+        scope = 2;
```

Comment:
notepad changes seem fine 
but I think scope = 1 should be ok for doc and photo

iirc there are still some other gear scripts that just use `scope == 2` checks and we really don't want these showing up